src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -3293,6 +3295,26 @@ class CreateRoleStatementSegment(BaseSegment):
         Ref("ObjectReferenceSegment"),
     )
 
+class DropRoleStatementSegment(BaseSegment):
+    """A `DROP ROLE` statement with CASCADE option.
+
+    https://docs.exasol.com/sql/drop_role.htm
+    """
+
+    type = "drop_role_statement"
+
+    is_ddl = False
+    is_dml = False
+    is_dql = False
+    is_dcl = True
+
```

Comment:
Only Exasol uses these (and not entirely sure they should! But that's another question).

```suggestion
```